pyproj error:

```
=================================== FAILURES ===================================                                                                                                                                                              │
│____________________________ test_datum__from_epsg _____________________________                                                                                                                                                              │
│                                                                                                                                                                                                                                              │
│    def test_datum__from_epsg():                                                                                                                                                                                                              │
│        datum_wkt = (                                                                                                                                                                                                                         │
│            'ENSEMBLE["World Geodetic System 1984 ensemble",'                                                                                                                                                                                 │
│            'MEMBER["World Geodetic System 1984 (Transit)",ID["EPSG",1166]],'                                                                                                                                                                 │
│            'MEMBER["World Geodetic System 1984 (G730)",ID["EPSG",1152]],'                                                                                                                                                                    │
│            'MEMBER["World Geodetic System 1984 (G873)",ID["EPSG",1153]],'                                                                                                                                                                    │
│            'MEMBER["World Geodetic System 1984 (G1150)",ID["EPSG",1154]],'                                                                                                                                                                   │
│            'MEMBER["World Geodetic System 1984 (G1674)",ID["EPSG",1155]],'                                                                                                                                                                   │
│            'MEMBER["World Geodetic System 1984 (G1762)",ID["EPSG",1156]],'                                                                                                                                                                   │
│            'MEMBER["World Geodetic System 1984 (G2139)",ID["EPSG",1309]],'                                                                                                                                                                   │
│            'ELLIPSOID["WGS 84",6378137,298.257223563,LENGTHUNIT["metre",1],'                                                                                                                                                                 │
│            'ID["EPSG",7030]],ENSEMBLEACCURACY[2.0],ID["EPSG",6326]]'                                                                                                                                                                         │
│        )                                                                                                                                                                                                                                     │
│>       assert Datum.from_epsg("6326").to_wkt() == datum_wkt                                                                                                                                                                                  │
│E       assert 'ENSEMBLE["Wo..."EPSG",6326]]' == 'ENSEMBLE["Wo..."EPSG",6326]]'                                                                                                                                                               │
│E                                                                                                                                                                                                                                             │
│E         Skipping 464 identical leading characters in diff, use -v to show                                                                                                                                                                   │
│E         - G",1309]],ELLIPSOID["WGS 84",6378137,298.257223563,LENGTHUNIT["metre",1],ID["EPSG",7030]],ENSEMBLEACCURACY[2.0],ID["EPSG",6326]]                                                                                                  │
│E         + G",1309]],MEMBER["World Geodetic System 1984 (G2296)",ID["EPSG",1383]],ELLIPSOID["WGS 84",6378137,298.257223563,LENGTHUNIT["metre",1],ID["EPSG",7030]],ENSEMBLEACCURACY[2.0],ID["EPSG",6326]]                                     │
│E         ?        +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++                                                                                                                                                              │
│                                                                                                                                                                                                                                              │
```

--- a/pkgs/development/python-modules/pyproj/default.nix
+++ b/pkgs/development/python-modules/pyproj/default.nix
@@ -3,17 +3,17 @@
   buildPythonPackage,
   fetchFromGitHub,
   fetchpatch,
-  python,
-  proj,
+  pytestCheckHook,
   pythonOlder,
   substituteAll,
-  cython,
-  pytestCheckHook,
-  mock,
+
   certifi,
+  cython,
+  mock,
   numpy,
-  shapely,
   pandas,
+  proj,
+  shapely,
   xarray,
 }:
 
@@ -51,11 +51,11 @@ buildPythonPackage rec {
   propagatedBuildInputs = [ certifi ];
 
   nativeCheckInputs = [
-    pytestCheckHook
     mock
     numpy
-    shapely
     pandas
+    pytestCheckHook
+    shapely
     xarray
   ];
 

--- a/pkgs/development/python-modules/pyproj/default.nix
+++ b/pkgs/development/python-modules/pyproj/default.nix
@@ -2,6 +2,7 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  fetchpatch,
   python,
   proj,
   pythonOlder,
@@ -35,6 +36,12 @@ buildPythonPackage rec {
       src = ./001.proj.patch;
       proj = proj;
       projdev = proj.dev;
+    })
+
+    # fix test failure caused by update of EPSG DB
+    (fetchpatch {
+      url = "https://github.com/pyproj4/pyproj/commit/3f7c7e5bcec33d9b2f37ceb03c484ea318dff3ce.patch";
+      hash = "sha256-0J8AlInuhFDAYIBJAJ00XbqIanJY/D8xPVwlOapmLDE=";
     })
   ];
 

